client/src/elm/Pages/WellChildEncounter/View.elm

```diff
@@ -165,7 +165,7 @@ viewPersonDetails language currentDate person =
         dateOfBirthEntry =
             Maybe.map
                 (\birthDate ->
-                    viewEntry Translate.DateOfBirth (formatDDMMYY birthDate)
+                    viewEntry Translate.DateOfBirth (formatDDMMyyyy birthDate)
```

Comment:
Please add screenshots for changes at this file.
Since date becomes longer, we may need to adjust the styling.

Reply:
![Screen Shot 2021-12-01 at 15 12 02](https://user-images.githubusercontent.com/32199791/144241204-d45330c2-0abe-4a16-9692-04b17ec0e8a2.png)